src/views/components/top.pug

```diff
@@ -1,14 +1,57 @@
 .top-wrapper.js-section
+  //- script(type="application/javascript", async="async", src="https://icoholder.com/en/widget/big-green/21042.js?width=300")
   .top
     h1.top__title.title-lg.h-text-gradient!=app.top.title
-    .top__beta-btn
-      a.beta-btn(href="https://beta.membrana.io/" target="_blank")
-        img.beta-btn__icon.js-prevent-inline(src="/static/img/icons/beta.svg" alt="")
-        span.beta-btn__text.h-uppercase!=app.top.beta_text
-    .top__description.h-uppercase!=app.top.description
-    .top__socials
-      .socials
-        include ../shared/socials
-        +socials('top')
+    h3.top__title2
+      span.top__title2-white PRESALE  
+      span.top__title2-col STARTING SOON
+    a(href="").top__btn
+      img(src="/static/img/icons/play2.svg" alt="").top__btn_img
+      span.top__btn_span Registration will open soon
+      //- span.top-bold.top__btn_span open SOON
+    .top__timer.top_timer
+      p.top_timer__desc
+        span.top-bold 90% bonus 
+        | for 24H starts in
+      .top_timer__wr
+        .top_timer__d
+          span.top-bold_700.timerJs 28
+          | d
+          span.top_timer-colored :
+        .top_timer__h
+          span.top-bold_700.timerJs 22
+          | h
+          span.top_timer-colored :
+        .top_timer__m
+          span.top-bold_700.timerJs 59
+          | m
+    .top__widgets
+      // a(href="https://icomarks.com/ico/membrana", target="_blank", rel="nofollow", title="Membrana ICO").top__widget1.top__widget
```

Comment:
Another unknown comment.

Reply:
this widget was layout-like but after u've asked to change it for another one. I'm not sure it wont be used again so i just saved it like this. If no feel free to remove it